README.md

```diff
@@ -10,18 +10,17 @@ Obtain an access TOKEN from [stattleship.com](www.stattleship.com). Load R and i
 
 ```
 library(stattleshipR)
-TOKEN <- 'insert-your-token-here'
+setToken('insert-your-token-here')
 ```
 
 Get all NBA players:
 
 ```
-library(stattleshipR)
 league = "nba"
 sport = "basketball"
 ep = "players"
 q_body = list()
-players = stattle(TOKEN, sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
+players = stattle(sport=sport, league=league, ep=ep, query=q_body, version=1, verbose=TRUE, walk=TRUE)
```

Comment:
I think it's be great in subsequent versions to make the calls:

`basketball_players(query=q_body)`
`basketball_teams(query=q_body)`
`football_teams(query=q_body)`

and let the "stattle" or "perform" get used in endpoint specific calls.